kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/shared/model/GwtWiresChannelType.java

```diff
@@ -13,7 +13,7 @@
 package org.eclipse.kura.web.shared.model;
 
 /**
- * This enum corresponds to the one provided in {@link org.eclipse.kura.asset.ChannelType}
+ * This enum corresponds to the one provided in {@code org.eclipse.kura.channel.ChannelType}
```

Comment:
If i use @link Eclipse forces me to import org.eclipse.kura.channel.ChannelType and this causes compilation problems because this source is also compiled by GWT.
By the way I don't think this is a big issue because this is an implementation class.